hack/dockerfiles/lint.Dockerfile

```diff
@@ -20,6 +20,13 @@ RUN --mount=target=/go/src/github.com/moby/buildkit --mount=target=/root/.cache,
   yamllint -c .yamllint.yml --strict . && \
   touch /yamllint.done
 
+FROM yoheimuta/protolint as protolint
```

Comment:
can we pin this to a specific version (perhaps with a build-arg to override)? it looks like they publish versions; https://hub.docker.com/r/yoheimuta/protolint/tags

I noticed that their Dockerfile i alpine-based, and does an `apk upgrade`; https://hub.docker.com/layers/yoheimuta/protolint/0.45.0/images/sha256-3c047a5e7714123141efbdc3879cbae12ef5bcc5ab869c99e50af043eb8070b9?context=explore

But, AFAICS, it only contains a single (static?) binary; should we consider a `COPY --from` to only use the binary, and not the rest of the image?

Reply:
> I noticed that their Dockerfile i alpine-based, and does an `apk upgrade`; https://hub.docker.com/layers/yoheimuta/protolint/0.45.0/images/sha256-3c047a5e7714123141efbdc3879cbae12ef5bcc5ab869c99e50af043eb8070b9?context=explore
> 
> But, AFAICS, it only contains a single (static?) binary; should we consider a `COPY --from` to only use the binary, and not the rest of the image? Or alternatively take binaries from https://github.com/yoheimuta/protolint/releases

I think we can do both. Using the image and bind mount the binary like: https://github.com/crazy-max/diun/blob/fe0622ecac59190d55eeea5f6a88f05a885300a3/hack/lint.Dockerfile#L11-L15

Reply:
Done.